src/Sinch/Core/Utils.cs

```diff
@@ -72,7 +72,7 @@ public static T ParseEnum<T>(string value)
                 }
             }
 
-            throw new InvalidOperationException($"Failed to parse {nameof(T)} enum");
+            throw new InvalidOperationException($"Failed to parse {enumType.Name} enum");
```

Comment:
Does this mean any unknown value will trigger an exception ?
My thoughts from https://github.com/sinch/sinch-sdk-internal-docs/blob/main/ADR/0005-enums.md was we should be able to handle new values *dynamically*: 
> User is able to create new values if needed.
Deserialization is able to create new instances if needed.

Currently, we will have different behavior between `.net` and `java` SDKs (https://github.com/sinch/sinch-sdk-java-proto/pull/2)

Reply:
Yeah, this is intended. Currently I want it to be left as it is. We can revisit it latter nonetheless.